### DISCUSSION TITLE
Add the ability to Hide models

### DISCUSSION THREAD

**Discussion by JustMaier** [civitai/civitai#65] [Ideas]
Give users the ability to hide models from the main feed. This would allow them to kind of manually curate their main page. We could also then use the data to even offer to auto curate for them.

**Comment by JustMaier** [comment]
We've decided to currently do this at the user level.

Here's what it looks like:
![Example](https://user-images.githubusercontent.com/607609/208995544-74c26017-b2f4-4d85-92f1-0b28f1813e46.png)

**Reply by Seantourage** [reply]
Thanks so much!
